playbooks/ansible/roles/test.sit-test-cases/templates/test-info.yml.j2

```diff
@@ -10,11 +10,13 @@ users:
     {%- endfor +%}
   {%- endfor +%}
 
-backend: {{ test_backend }}
+backend: {{ config.be.name }}
```

Comment:
We should consider removing this global backend valuesince this value will be overridden by the backend-name for each share later.  But this is not a high prio requirement and it may be useful when adding multiple variants in the same test.